src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingCoursesViewModel.cs

```diff
@@ -23,5 +29,71 @@ public void Then_The_Total_Message_Is_Created_Correctly(int totalCount,int filte
 
             viewModel.TotalMessage.Should().Be(expectedMessage);
         }
+
+        [Test, AutoData]
+        public void Then_No_Filter_Items_Builds_Correct_Clear_Links()
+        {
+            //Arrange
+            var viewModel = new CoursesViewModel();
+            
+            //Assert
+            viewModel.ClearKeywordLink.Should().BeEmpty();
+            viewModel.ClearSectorLinks.Should().BeEmpty();
+        }
+
+        [Test, AutoData]
+        public void Then_The_Clear_Keyword_Link_Is_Generated_If_Filtered_By_Keyword_With_Sectors(List<Guid> selectedRoutes, string keyword)
+        {
+            //Arrange Act
+            var model = BuildCoursesViewModel(selectedRoutes, keyword);
+
+            //Assert
+            Assert.IsNotNull(model.ClearKeywordLink);
+            Assert.AreEqual("?sectors=" + string.Join("&sectors=", model.SelectedSectors), model.ClearKeywordLink);
+        }
+        
+        [Test, AutoData]
+        public void Then_The_Clear_Filter_Items_Are_Built_From_The_Selected_Items(List<Guid> selectedRoutes, string keyword)
+        {
+            //Arrange Act
+            var model = BuildCoursesViewModel(selectedRoutes, keyword);
+
+            //Assert
+            var clearLinkCount = selectedRoutes.Count;
+            Assert.AreEqual(clearLinkCount,model.ClearSectorLinks.Count);
+            
+            foreach (var selectedRoute in selectedRoutes)
+            {
+                var sector = model.Sectors.SingleOrDefault(c => c.Id.Equals(selectedRoute));
+                
+                Assert.IsNotNull(sector);
+                Assert.IsTrue(model.ClearSectorLinks.ContainsKey(sector.Route));
+                Assert.AreEqual(clearLinkCount-1, model.ClearSectorLinks.Count(c=>c.Value.Contains($"sectors={selectedRoute}")));    
+                Assert.AreEqual(clearLinkCount, model.ClearSectorLinks.Count(c=>c.Value.Contains($"keyword={keyword}")));    
+            }
+
+            
+        }
+
```

Comment:
I think there's another test for keyword but no sectors.

Reply:
Added test to cover this